processes/executive-copy-principles.md

```diff
@@ -0,0 +1,38 @@
+# Executive Copy Principles
+
+## Clarity
+
+As the entity responsible for producing the executive copy, you must ensure:
+* That the effects and impacts of the proposal if passed are made as clear as possible to voters.
+* That the processes that have led to each item being included in the executive vote are made as transparent and easily understood as possible.
+* That the identities (can be pseudonymous or at an entity level) of those responsible for publishing, writing, and reviewing the executive are as transparent as possible. 
+
+The overall goal here is to eliminate ambiguity and confusion as much as reasonably possible. In an ideal world, voters should never need to ask or wonder why an item has been included in an executive, what will happen if it passes, or which entities are responsible for the executive and its contents.
+
+Clarity is important because it reduces the risk of disruption, drama, and a breakdown of legitimacy around the governance process. 
+
+## Informed Voters
+
+As the entity responsible for producing the executive copy, you must ensure that voters have access to sufficient information to vote (or not) in an informed way. This might include:
```

Comment:
```suggestion
As the entity responsible for producing the executive copy, you must ensure that voters have access to sufficient information to vote (or not) in an informed way. This might include:

```

And so on.